portafly/src/components/data-list/modals/ChangeStateModal.tsx

```diff
@@ -55,10 +55,10 @@ const ChangeStateModal: React.FunctionComponent<Props> = ({
 
   const options = [
     { value: '', label: '' },
-    { value: 'approved', label: t('states.approved') },
-    { value: 'pending', label: t('states.pending') },
-    { value: 'rejected', label: t('states.rejected') },
-    { value: 'suspended', label: t('states.suspended') }
+    { value: 'approved', label: t('audienceAccountsListing:actions_filter_options.by_state_options.approved') },
+    { value: 'pending', label: t('audienceAccountsListing:actions_filter_options.by_state_options.pending') },
+    { value: 'rejected', label: t('audienceAccountsListing:actions_filter_options.by_state_options.rejected') },
+    { value: 'suspended', label: t('audienceAccountsListing:actions_filter_options.by_state_options.suspended') }
```

Comment:
We'll need to extract this options array and pass it down

Reply:
The idea was that these are fixed options, no?

Reply:
If those are gonna be common and "fixed", then they shouldn't be within `audienceAccountsListing` namespace but one less specific

Reply:
Yeah, I think that's the error. We should discuss that further in another PR, since this is a bug fix.

Reply:
I moved them in the end 😓